packages/components/src/date-time/README.md

```diff
@@ -1,6 +1,14 @@
 # DateTimePicker
 
-DateTimePicker is a React component to render a calendar and clock for selecting a date and time. The calendar and clock components can be accessed individually using the `DatePicker` and `TimePicker` components respectively.
+DateTimePicker is a React component that renders a calendar and clock for date and time selection. The calendar and clock components can be accessed individually using the `DatePicker` and `TimePicker` components respectively.
+
+![Date Time component](https://wordpress.org/gutenberg/files/2019/07/date-time-picker.png)
+
+## Best practices
+
+Date pickers should:
+
+- Use smart defaults and highlight the current date.
```

Comment:
Isn't this a behavior that is already absorbed by the component itself? Is it worth adding to the docs? A user might ask "how do I do that" since it's mentioned in the "best practices"?

Reply:
I see where you're coming from. The idea behind documenting this is to help define the best practices for a component like this in terms of functionality. I think it's useful in case folks in the future want to make an update to this component that might affect this functionality as well as a precedent in case future folks want to make a new component that has similar functionality.